src/instruments/src/ND/elements/RadioNavInfo.tsx

```diff
@@ -11,11 +11,11 @@ export enum NavAidMode {
 
 export type RadioNavInfoProps = { index: 1 | 2, side: EfisSide }
 
-const TuningModeIndicator: React.FC<{ index: 1 | 2, frequency: number }> = ({ index, frequency }) => {
-    const [tuningMode] = useSimVar(`L:A32NX_FMGC_RADIONAV_${index}_TUNING_MODE`, 'enum');
+const TuningModeIndicator: React.FC<{ index: 1 | 2 }> = ({ index }) => {
```

Comment:
This needed to remain per-side. VOR1 can be manually tuned while VOR2 is auto-tuned for example.